.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature Request
+about: Suggest a new feature for the rippled project
+title: "[Title with short description] (Version: [rippled version])"
+labels: Feature Request
+assignees: ''
+
+---
+<!-- Please search existing issues to avoid creating duplicates.-->
+
+## Description
+### Describe the feature you'd like
+<!-- Be as detailed as possible here.-->
+
+### Describe The rationale
+<!-- Why do we need this feature?-->
+
+### Describe the Solution
+<!-- What is the solution?-->
+
+### Describe path not taken
```

Comment:
I would rename this heading to Alternatives or Paths Not Taken.

Reply:
Will do